falcon/request.py

```diff
@@ -838,6 +838,43 @@ def get_param_as_list(self, name,
 
         raise HTTPMissingParam(name)
 
+    def get_param_as_date(self, name, format_string='%Y-%m-%d',
+                          required=False, store=None):
+        """
+        Args:
+            name (str): Parameter name, case-sensitive (e.g., 'ids').
+            format_string (str): String used to parse the param value into a date. # NOQA
+                See https://docs.python.org/3.4/library/datetime.html#strftime-and-strptime-behavior #NOQA
```

Comment:
For this line, we can replace the link with something like:

"Any format recognized by strptime() is supported."

That would give the reader enough information to go look it up on their own if they need to.